src/main/viz/GenomeTrack.js

```diff
@@ -53,7 +53,7 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
         // We only push objects in the text case as it involves creating a
         // new object & can become a performance issue.
         // 0.5 = centered
-        ctx.fillText(letter, scale(1 + 0.5 + pos), height - 1);
+        ctx.fillText(letter, scale(1 + 0.5 + pos), height / 2);
```

Comment:
Ahh. I've tried it out and it actually seems fine. Using height/2 places the text centered as we would like it:
<img width="878" alt="screen shot 2016-11-10 at 2 17 02 pm" src="https://cloud.githubusercontent.com/assets/8368618/20196846/69b77e0c-a751-11e6-8e37-f60671d45573.png">